modules/core/resources/META-INF/smithy/documentation.smithy

```diff
@@ -2,8 +2,8 @@ $version: "2"
 
 namespace alloy
 
-@trait(selector: "operation")
 /// A version of @examples that is not tied to a validator
```

Comment:
This is what was happening before:

<img width="1919" alt="image" src="https://github.com/disneystreaming/alloy/assets/894884/1402a4d7-2fd2-4744-b20b-28f76739a5f9">